src/PrestaShopBundle/Resources/config/services/adapter/module.yml

```diff
@@ -89,8 +89,15 @@ services:
   prestashop.adapter.legacy.module:
     class: PrestaShop\PrestaShop\Adapter\Module\Module
 
-  prestashop.adapter.module.repository.module_repository:
-    class: 'PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository'
+  PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository:
+    public: false
     arguments:
       - !php/const _PS_ROOT_DIR_
       - !php/const _PS_MODULE_DIR_
+
+  prestashop.adapter.module.repository.module_repository:
+    alias: 'PrestaShop\PrestaShop\Adapter\Module\Repository\ModuleRepository'
+    deprecated:
+      package: PrestaShop\PrestaShop
+      version: 9.0
+    public: true
```

Comment:
Isn't `true` by default in this file and so this line could be useless? 🤔

Reply:
Probably but I wanted to be explicit since the alias class is private just in case